pkg/network/controllers/hotplug.go

```diff
@@ -66,3 +66,14 @@ func ClearDetachedInterfaces(
 
 	return ifaces, vmispec.FilterNetworksByInterfaces(specNets, ifaces)
 }
+
+func UpdateInterfaceState(filteredVMIIfaces, vmIfaces []v1.Interface) {
```

Comment:
Is it possible to extend `ApplyDynamicIfaceRequestOnVMI` [1] to support the desired logic? 

[1] https://github.com/kubevirt/kubevirt/blob/1e817cb894edb98a3c567566769632756e873fea/pkg/network/controllers/hotplug.go#L28